tests/app/main/test_frameworks.py

```diff
@@ -5353,7 +5050,7 @@ def test_api_is_not_called_and_no_email_sent_for_subsequent_posts(self, mandrill
         )
         assert res.status_code == 200
         assert self.data_api_client.agree_framework_variation.called is False
-        assert mandrill_send_email.called is False
+        assert mocked_notify_client.called is False
```

Comment:
Great 👍 Though I've just realised the happy path test above doesn't assert that the Notify client is actually called (I missed that one first time round, sorry).

Reply:
Where sorry?